kolibri/plugins/facility/assets/src/views/ManageClassPage/index.vue

```diff
@@ -92,6 +101,72 @@
         @cancel="closeModal"
         @success="handleCreateSuccess()"
       />
+
+      <ClassRenameModal
+        v-show="openRenameModal"
```

Comment:
Lets use v-if instead of v-show here, to avoid rendering unnecessary nodes to the dom.

Reply:
Although this seems to be affecting the keyboard navigation on the dropdown menu.